src/vivarium_ciff_sam/constants/scenarios.py

```diff
@@ -20,19 +22,40 @@ def __init__(
         self.has_lbwsg = has_lbwsg
 
 
-class __Scenarios(NamedTuple):
-    BASELINE: Scenario = Scenario('baseline')
-    WASTING_TREATMENT: Scenario = Scenario('wasting_treatment', has_alternative_treatment=True)
-    SQLNS: Scenario = Scenario('sqlns', has_alternative_treatment=True, has_sqlns=True)
-    LBWSG_INTERVENTIONS: Scenario = Scenario(
+class __InterventionScenarios(NamedTuple):
+    BASELINE: InterventionScenario = InterventionScenario('baseline')
+    WASTING_TREATMENT: InterventionScenario = InterventionScenario('wasting_treatment', has_alternative_treatment=True)
+    SQLNS: InterventionScenario = InterventionScenario('sqlns', has_alternative_treatment=True, has_sqlns=True)
+    LBWSG_INTERVENTIONS: InterventionScenario = InterventionScenario(
         'lbwsg_interventions', has_alternative_treatment=True, has_sqlns=True, has_lbwsg=True
     )
 
-    def __getitem__(self, item) -> Scenario:
+    def __getitem__(self, item) -> InterventionScenario:
+        for scenario in self:
+            if scenario.name == item:
+                return scenario
+        raise KeyError(item)
+
+
+INTERVENTION_SCENARIOS = __InterventionScenarios()
+
+
+class SamKScenario:
```

Comment:
what is this / why is it named sam k?

Reply:
This is governing the new scenario branch for sam k (the k is referring to a k-factor that governs differential remissions due to treatment)/